Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -221,149 +262,258 @@ public List<InteractableEvent> InteractableEvents
             }
         }
 
+        private List<InteractableThemeBase> activeThemes = new List<InteractableThemeBase>();
+
         /// <summary>
         /// The list of running theme instances to receive state changes
         /// When the dimension index changes, the list of themes that are updated changes to those assigned to that dimension.
         /// </summary>
-        private List<InteractableThemeBase> activeThemes = new List<InteractableThemeBase>();
+        public IReadOnlyList<InteractableThemeBase> ActiveThemes => activeThemes.AsReadOnly();
 
-        // directly manipulate a theme value, skip blending
-        protected bool forceUpdate = false;
+        /// <summary>
+        /// How many times this interactable was clicked
+        /// </summary>
+        /// <remarks>
+        /// Useful for checking when a click event occurs.
+        /// </remarks>
+        public int ClickCount { get; private set; }
 
```

Comment:
this isn't really doing anything .. can be deprecated.

Reply:
we are not using but someone might be. It's pretty harmless so I'd rather not deprecate now.

Reply:
it doesn't feel right because the purpose of interactable is provide a state machine by processing all inputs. I don't see why we can't mark this as obsolete. If someone wants this functionality they have always pipe it by creating a click event receiver handler



---
In reply to: [329202629](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6104#discussion_r329202629) [](ancestors = 329202629)

Reply:
A ClickCount is state information. Yes someone could go recreate the functionality themselves but why make them do that? By deprecating we are more likely to do harm then good.

What is the benefit to deprecating this? What do we gain?

Reply:
Going to resolve this as won't fix to
1) move forward on the PR
2) adhere to the breaking change guidelines of MRTK

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/BreakingChanges.html